oracle/datadog_checks/oracle/oracle.py

```diff
@@ -164,20 +165,29 @@ def _get_custom_metrics(self, con, custom_queries, global_tags):
 
             with closing(con.cursor()) as cursor:
                 cursor.execute(query)
-                row = cursor.fetchone()
-                if row:
+                rows = cursor.fetchall()
```

Comment:
Can we instead iterate through cursor to avoid putting everything in memory?

Reply:
We can. I copied from the postgres integration, I assume it makes things faster.

Reply:
Looks like I copied the wrong part, fixing it.